.azure-pipelines/powershell-core.yml

```diff
@@ -17,6 +17,7 @@ variables:
   AnalysisTimeoutInMinutes: 120
   TestTimeoutInMinutes: 180
   BuildAzPredictor: false
+  Azure_PS_TestCoverage: true
```

Comment:
I think it will be better if the name can align with other variables such as `EnableAzurePSTestCoverage` or just `EnableTestCoverage`

Reply:
This is an environment variable and actually the naming convention is from the config framework. I will update to match with the convention for the CI